app/src/main/java/com/example/puzzlepic/ui/main/MainFragment.kt

```diff
@@ -76,55 +83,56 @@ class MainFragment : Fragment() {
         requestCode: Int,
         permissions: Array<out String>,
         grantResults: IntArray
-    ){
+    ) {
         super.onRequestPermissionsResult(requestCode, permissions, grantResults)
-        when(requestCode){
+        when (requestCode) {
             CAMERA_PERMISSION_REQUEST_CODE -> {
-                if(grantResults.isNotEmpty() && grantResults[0] == PackageManager.PERMISSION_GRANTED){
+                if (grantResults.isNotEmpty() && grantResults[0] == PackageManager.PERMISSION_GRANTED) {
                     takePhoto()
                 } else {
-                    Toast.makeText(context, "Unable to take photo without permission", Toast.LENGTH_LONG).show()
+                    Toast.makeText(
+                        context,
+                        "Unable to take photo without permission",
+                        Toast.LENGTH_LONG
+                    ).show()
                 }
             }
         }
     }
 
-    private fun takePhoto(){
-        Intent(MediaStore.ACTION_IMAGE_CAPTURE).also{
-            takePictureIntent -> takePictureIntent.resolveActivity(requireContext().packageManager)?.also{
+    private fun takePhoto() {
+        Intent(MediaStore.ACTION_IMAGE_CAPTURE).also { takePictureIntent ->
+            takePictureIntent.resolveActivity(requireContext().packageManager)?.also {
                 startActivityForResult(takePictureIntent, CAMERA_REQUEST_CODE)
-        }
+            }
         }
     }
 
-    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?){
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
         super.onActivityResult(requestCode, resultCode, data)
-        if(resultCode == RESULT_OK){
-            if(requestCode == CAMERA_REQUEST_CODE){
+        if (resultCode == RESULT_OK) {
+            if (requestCode == CAMERA_REQUEST_CODE) {
                 val imageBitmap = data!!.extras!!.get("data") as Bitmap
                 image1.setImageBitmap(imageBitmap)
             }
         }
 
     }
 
-
     /**
      * This will set the center photo box with a photo from the internet when the "Get Random Puzzle Button" is selected
      */
-    private fun testRandomPhoto(randomURL : String = "") {
-         val Image_Url = randomURL
+    private fun testRandomPhoto(randomURL: String = "") {
         val imageView = image5
         Picasso
             .with(context)
-            .load(Image_Url)
-            .resize(300,300)
+            .load(randomURL)
```

Comment:
The camelCasing is correct... though I'd make the variable name descriptive.  imageUrl or imageURL works.